docs/user-guide/migrate-to-greptimedb/migrate-from-mysql.md

```diff
@@ -0,0 +1,94 @@
+# Migrate from MySQL
+
+This document will guide you through the migration process from MySQL to GreptimeDB.
+
+## Before you start the migration
+
+Please be aware that though GreptimeDB supports the wire protocol of MySQL, it does not mean GreptimeDB implements all
+MySQL's features. You may refer to the "[ANSI Compatibility](/reference/sql/compatibility.md)" to see the
+constraints regarding using SQL in GreptimeDB.
+
+## Migration steps
+
+### Create the databases and tables in GreptimeDB
+
+Before migrating the data from MySQL, you first need to create the corresponding databases and tables in GreptimeDB.
+GreptimeDB has its own SQL syntax for creating tables, so you cannot directly reuse the table creation SQLs that are produced
+by MySQL.
+
+When you write the table creation SQL for GreptimeDB, it's important to understand
+its "[data model](/user-guide/concepts/data-model.md)" first. Then, please take the following considerations in
+your create table SQL:
+
+1. Since the time index column cannot be changed after the table is created, you need to choose the time index column carefully.
```

Comment:
Let's give some examples. For example,  the user can use the collection time of sensor data or the occurrence time of an event as the time index.

Reply:
c [7d82b2b](https://github.com/GreptimeTeam/docs/pull/1241/commits/7d82b2b603ffe893e30e7cff8df524627415453e)